Physics/ContactCallbacks/IContactEventHandler.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright The Authors of bepuphysics2
```

Comment:
maybe keep all bepu files in the bepu folder

Reply:
addressed above